src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/CodeActions/CodeActionEndpoint.cs

```diff
@@ -298,7 +299,15 @@ internal async Task<RazorVSInternalCodeAction[]> GetCodeActionsFromLanguageServe
             LanguageKind = languageKind
         };
 
-        return await _languageServer.SendRequestAsync<DelegatedCodeActionParams, RazorVSInternalCodeAction[]>(RazorLanguageServerCustomMessageTargets.RazorProvideCodeActionsEndpoint, delegatedParams, cancellationToken);
+        try
+        {
+            return await _languageServer.SendRequestAsync<DelegatedCodeActionParams, RazorVSInternalCodeAction[]>(RazorLanguageServerCustomMessageTargets.RazorProvideCodeActionsEndpoint, delegatedParams, cancellationToken).ConfigureAwait(false);
+        }
+        catch (RemoteInvocationException e)
+        {
+            logger.LogException(e, "Error getting code actions from delegate language server for {languageKind}", languageKind);
```

Comment:
This means we'll get a prism bug if enough of these happens, which will be good to highlight for partner teams, but won't prevent us from showing Razor code actions either